src/Transport/HttpTransport.php

```diff
@@ -108,24 +119,41 @@ public function send(Event $event): ?string
         } else {
             try {
                 $promise->wait();
-            } catch (\Exception $exception) {
+            } catch (\Throwable $exception) {
                 return null;
```

Comment:
👍 return `null` in case we are not able to send the event is what we also do in other SDKs.